### ISSUE TITLE
Bug with calculating the Training Loss

### ISSUE THREAD

**Issue by sashkboos** [AshwinRJ/Federated-Learning-PyTorch#19]
https://github.com/AshwinRJ/Federated-Learning-PyTorch/blob/235b8f0ab161825694ee73874295d773b0d3f23f/src/federated_main.py#L105

Here, I think we have to replace the idx with _c_ because we want to calculate the Training error of the global_model on all training data (after the averaging).

**Comment by heymyNLI** [comment]
I agree with sashkboos

**Comment by bigmoumou** [comment]
Agree. It should loop all the clients.

**Comment by wangclin** [comment]
of couse, it is a mistake apparently
